Utils.py

```diff
@@ -515,7 +515,7 @@ def filter(self, record: logging.LogRecord) -> bool:
             return self.condition(record)
 
     file_handler.addFilter(Filter("NoStream", lambda record: not getattr(record,  "NoFile", False)))
```

Comment:
```suggestion
    file_handler.addFilter(Filter("NoStream", lambda record: not getattr(record, "NoFile", False)))
```
Can we fix this just because it's nearby?

Reply:
Extreme bruh moment but sure you've already written it out for me lol